### ISSUE TITLE
Add the --no-track option to bosh cli v2

### ISSUE THREAD

**Issue by holgero** [cloudfoundry/bosh-cli#266]
Hi,

bosh cli v1 had an option `--no-track` (`-N`) that avoided tailing the log of the task and only printed the task id. I would like to have that option also in bosh cli v2.

My use case is this: I would like to have concourse updating itself. For that to work, the job triggering the update should finish without waiting for the update, so that the concourse-worker which runs the job can be drained.

**Comment by cppforlife** [comment]
@holgero let me think about this for a little bit. it's an interesting use case...

**Comment by cppforlife** [comment]
@ronakbanka are you using it in concourse as well?

**Comment by ronakbanka** [comment]
@cppforlife Not concourse but something similar use case, running a testing errand from the script and move to next step instead of waiting for the exit code.

**Comment by holgero** [comment]
Well are there any strong reasons to not implement that `--no-track` option in bosh cli v2? Or is it just a question of prioritization? Or in other words: How likely would a pull request that implements the option be merged?

**Comment by holgero** [comment]
bump

**Comment by dpb587-pivotal** [comment]
Some of the trickier scenarios is that a single bosh invocation could result in multiple tasks being executed. For example, the `bosh deploy` command might uploading multiple releases (each with their own task, possibly in parallel) before a deployment (which probably is the task you're interested in). In the future, errands or other one-off commands may also result in multiple tasks.

Due to those potential complications, there's not any work planned around this yet. Hopefully it explains a bit more about the problem space.

Have any ideas around alternatives, or recommendations on ways other tools have solved the parallel+multiple+chained `--follow-task`-type issue?

**Comment by holgero** [comment]
So the bosh cli is orchestrating these tasks? This makes it more difficult, of course. Has the cli a chance to know when it has started the last task on the director in the chain? It could terminate at that point and not wait for the last task to finish. This would probably work in the concourse scenario, but seems to be a bit odd as an command line option...

How did it work in bosh cli version one, by the way? It had this --no-track option.

**Comment by pcrocker-pivotal** [comment]
+1

I have [a stop/start script](https://github.com/patrickcrocker/pcf-stuff/blob/master/boshctl) that loops through all my `bosh deployments` and runs `bosh -n -N stop --hard` (or `bosh -n -N start`) on each deployment.  I'm working on refactoring it to use the bosh 2 cli, but I need the `--no-track` option.

**Comment by ChaosEternal** [comment]
@dpb587-pivotal 
I suggest this solution: 
when a deploy command involves multiple tasks like uploading releases, it just can not be "--no-track"-ed, 
users will be forced to upload release using separate task.

**Comment by freddesbiens** [comment]
We acknowledge that there can be improvements in this area. However, given our current workload, we will not address this anytime soon since script authors can easily leverage third-party tools to parallelize operations. Given this, I will close this issue for now.
